### ISSUE TITLE
Provide an way to specify the "default" Background

### ISSUE THREAD

**Issue by wizardzloy** [NewSpring/react-storybook-addon-backgrounds#20]
is it possible to specify the default background to use like:
```js
backgrounds([
  { name: "twitter", value: "#00aced", default: true },
  { name: "facebook", value: "#3b5998" },
])
```

**Comment by jbaxleyiii** [comment]
@wizardzloy you bet! I'll add that!

**Comment by jbaxleyiii** [comment]
@wizardzloy published as `0.0.6`

**Comment by wizardzloy** [comment]
@jbaxleyiii man, the speed of your reaction is crazy! Thank you!

**Comment by wizardzloy** [comment]
@jbaxleyiii a small issue I've noticed is that the "default" background is not automatically applied when I am loading the page. I use `backgrounds` as a "global" decorator, i.e. call `addDecorator(backgrounds([...]))` in my .storybook/config.js

**Comment by jbaxleyiii** [comment]
@wizardzloy yep! I just removed the extra swatch. Working on that now https://github.com/NewSpring/react-storybook-addon-backgrounds/issues/22

👍

**Comment by jbaxleyiii** [comment]
@wizardzloy published as `0.0.7`!

**Comment by lifehackett** [comment]
Thanks for adding support for default.  I am setting my background colors as a global decorator.  On the initial load, the default gets applied, but when switching between controls, it does not.
